### PR TITLE
[Update] auth/authen

### PR DESCRIPTION
**Pull request by dinhthi12**
# Related Tickets

- [#Ticket_ID](https://edu-redmine.vn/issues/xxxx)

## WHAT this PR does?

- ex: Change number items `completed/total` in admin page.

## Notes Impacted Areas

- N/A.

## Attachments

- N/A.

### CODE REVIEW

File: app/javascript/stylesheets/style.scss [hunk 1]
```diff
@@ -293,3 +293,46 @@
...
+
+  padding: 12px 16px 12px 42px;
+  border: none;
+  border-radius: 3px;
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, .04), 0 1px 1px rgba(0, 0, 0, .25);
+
+  color: #757575;
+  font-size: 14px;
+  font-weight: 500;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
```
Comment (hoangpt1310) [inline app/javascript/stylesheets/style.scss]: fix lại

File: app/views/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <div class="actions">
-    <%= f.submit "Send me reset password instructions" %>
-  </div>
-<% end %>
-
-<%= render "devise/shared/links" %>
+<div class="container">
```
Comment (hoangpt1310) [inline app/views/devise/passwords/new.html.erb]: đưa vào login.html.rb

File: app/views/devise/passwords/new.html.erb [hunk 3]
```diff
@@ -1,16 +1,44 @@
...
+                  <button class="btn btn-primary w-100" type="submit">Send me reset password instructions</button>
+                </div>
+              </form>
+              <div class="col-12 pt-4">
+                  <a type="btn" href="http://127.0.0.1:3000/" class="btn btn-info w-100" type="submit">Back to home</a>
+                </div>
+            </div>
+          </div>
+          <div class="credits">
+            Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
```
Comment (hoangpt1310) [inline app/views/devise/passwords/new.html.erb]: i18n

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,26 +1,61 @@
...
+                </div>
+                <div class="col-12">
+                  <button class="btn btn-primary w-100" type="submit">Login</button>
+                </div>
+                <div class="col-12">
+                  <p class="small mb-0">Don't have account? <%= render "devise/shared/links" %>
+                  </p>
+                </div>
+              </form>
+              <div class="google col-12">
```
Comment (hoangpt1310) [inline app/views/devise/sessions/new.html.erb]: fix google

**Comment by tn0810520** [comment]
fix tương tự

**Pull request merged**